great_expectations/render/renderer/site_builder.py

```diff
@@ -717,7 +709,9 @@ def _get_call_to_action_buttons(self, usage_statistics):
         return results
 
     # TODO: deprecate dual batch api support
-    def build(self, skip_and_clean_missing=True, build_index: bool = True):
+    def build(
+        self, skip_and_clean_missing: bool = True, build_index: bool = True
+    ) -> Tuple[Any, Optional[OrderedDict]]:
```

Comment:
This method was ~225 lines long so I split up the core behavior into helper functions. None of the underlying behavior is changed but this should hopefully make it easier to read.